graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -608,7 +651,15 @@ def _rename_or_suppress_or_ignore_name_and_add_to_record(
             self.renamed_to_builtin_scalar_conflicts[type_name] = desired_type_name
 
         self.reverse_name_map[desired_type_name] = type_name
+        # renamings is performing a 1:1 renaming so if renamings is iterable, record that this entry
+        # in renamings is used to ensure all renamings are used.
+        if isinstance(self.renamings, Iterable) and type_name in self.renamings:
```

Comment:
`self.renamed_types` is meant to contain the entries from the `renamings` mapping that are actually used (i.e. if `renamings` maps `A` to `B` _and_ `A` is in the schema, then `A` is to appear in `self.renamed_types`) and only meant to be used when `renamings` is iterable, to check that all the renamings were used. 

Upon re-reading this code, I found an easier way to do this (since we already have `self.reverse_name_map`, which isn't sufficient on its own because it doesn't contain suppressed types, but has all the renames) that also helps with the error message problems